src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5360,6 +5360,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_NoBestNullabilityConditionalExpression_Title" xml:space="preserve">
     <value>No best nullability for operands of conditional expression.</value>
   </data>
+  <data name="WRN_AnnotationDisallowedInObjectCreation" xml:space="preserve">
+    <value>Cannot use a nullable reference type in object creation.</value>
```

Comment:
Is there any reason this should not be an error?

Reply:
I'll change this to be an error.